_data/wai-evaluation-tools-list/submissions/2f4fe4d0-2799-11f0-accb-9944093ad21a.json

```
@@ -0,0 +1,42 @@
+{
+  "a11yloc": "https://a-plus.tech/accessibility-statement/",
+  "actrules": "",
+  "automated": [
+    "Component"
+  ],
+  "comments": "",
+  "contact": "accessibility@a-plus.tech",
+  "features": "This accessible tool evaluates color contrast based on both WCAG 2 and APCA methods.\r\n\r\nIt provides Live Preview with APCA-specific details, assisting messages, helpful explanations, and permalink support.\r\n\r\nAPCA is a new method of contrast evaluation, which is based on human perception of color. It is included in WCAG 3 drafts (as of publication).",
```

```suggestion
  "features": "This accessible tool evaluates color contrast based on both WCAG 2 and APCA methods.\r\n\r\nIt provides Live Preview with APCA-specific details, assisting messages, helpful explanations, and permalink support.\r\n\r\nAPCA is a new method of contrast evaluation, which is based on human perception of color.",
```

APCA method is not in the current [WCAG 3 Working Draft](https://www.w3.org/TR/2024/WD-wcag-3.0-20241212/).